src/threepp/input/PeripheralsEventSource.cpp

```diff
@@ -9,93 +9,3 @@ void PeripheralsEventSource::setIOCapture(IOCapture* capture) {
     ioCapture_ = capture;
 }
 
-void PeripheralsEventSource::addKeyListener(KeyListener& listener) {
-    auto find = std::find(keyListeners_.begin(), keyListeners_.end(), &listener);
-    if (find == keyListeners_.end()) {
-        keyListeners_.emplace_back(&listener);
-    }
-}
-
-bool PeripheralsEventSource::removeKeyListener(const KeyListener& listener) {
-    auto find = std::find(keyListeners_.begin(), keyListeners_.end(), &listener);
-    if (find != keyListeners_.end()) {
-        keyListeners_.erase(find);
-        return true;
-    }
-    return false;
-}
-
-void PeripheralsEventSource::addMouseListener(MouseListener& listener) {
-    auto find = std::find(mouseListeners_.begin(), mouseListeners_.end(), &listener);
-    if (find == mouseListeners_.end()) {
-        mouseListeners_.emplace_back(&listener);
-    }
-}
-
-bool PeripheralsEventSource::removeMouseListener(const MouseListener& listener) {
-    auto find = std::find(mouseListeners_.begin(), mouseListeners_.end(), &listener);
-    if (find != mouseListeners_.end()) {
-        mouseListeners_.erase(find);
-        return true;
-    }
-    return false;
-}
-
-void PeripheralsEventSource::onMousePressedEvent(int button, const Vector2& pos, PeripheralsEventSource::MouseAction action) {
-    if (ioCapture_ && ioCapture_->preventMouseEvent()) return;
```

Comment:
This feature has been removed in this work (preventing a input event to be fired in certain situations).